src/target/target_probe.h

```diff
@@ -48,6 +48,7 @@ bool stm32f1_probe(target_s *target);
 bool stm32f4_probe(target_s *target);
 bool stm32h5_probe(target_s *target);
 bool stm32h7_probe(target_s *target);
+bool stm32mp15_probe(target_s *target);
```

Comment:
This change needs mirroring in target_probe.c

Reply:
Mirrored the renamed probe in `_WEAK_NOP`'s.